ipld/get_shares.go

```diff
@@ -29,6 +29,8 @@ var NumConcurrentSquares = 8
 
 // Global worker pool that globally controls and limits goroutines spawned by
 // GetShares.
+// TODO(@Wondertan): Idle timeout for workers needs to be configured to around block time,
```

Comment:
Nope, unfortunately. The value is hardcoded in the lib and there is [no way to change it](https://github.com/gammazero/workerpool/issues/61). Also, the author of the lib seems unresponsive. I made [a PR](https://github.com/gammazero/workerpool/pull/59) almost a month ago and there was no feedback since then.